docs/operate/get-started/other-hardware/manage-modules.md

```diff
@@ -244,6 +244,24 @@ For more details, see the [`upload-module` GitHub Action documentation](https://
 
    Once the build is complete, the module will automatically update in the [Viam Registry](https://app.viam.com/registry), and the machines set to use the latest [version](/operate/reference/module-configuration/#module-versioning) of the module will automatically update to the new version.
 
+## Supported platforms for cloud-build
```

Comment:
https://deploy-preview-4336--viam-docs.netlify.app/operate/get-started/other-hardware/manage-modules/#supported-platforms-for-cloud-build